TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -206,12 +206,12 @@ static int CalculateCapacity(SeedOptions options)
 		void AddPresentItemLocations()
 		{
 			areaName = "Tutorial";
-			Add(ItemKey.TutorialMeleeOrb, "Yo Momma", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Melee));
-			Add(ItemKey.TutorialSpellOrb, "Yo Momma", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Spell));
+			Add(ItemKey.TutorialMeleeOrb, "Mommy's Orbs", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Melee)); //more descriptive while retaining the joke
+			Add(ItemKey.TutorialSpellOrb, "Mommy's Necklace", ItemProvider.Get(EInventoryOrbType.Blue, EOrbSlot.Spell)); //see above
 			areaName = "Lake Desolation";
-			Add(new ItemKey(1, 1, 1528, 144), "Starter chest 2", ItemProvider.Get(EInventoryUseItemType.FuturePotion), AccessToLakeDesolation);
-			Add(new ItemKey(1, 15, 264, 144), "Starter chest 3", ItemProvider.Get(EInventoryEquipmentType.OldCoat), AccessToLakeDesolation);
-			Add(new ItemKey(1, 25, 296, 176), "Starter chest 1", ItemProvider.Get(EInventoryUseItemType.FutureHiPotion), AccessToLakeDesolation);
+			Add(new ItemKey(1, 1, 1528, 144), "We are not in Kansas anymore", ItemProvider.Get(EInventoryUseItemType.FuturePotion), AccessToLakeDesolation); //these aren't always the Starter Chests
```

Comment:
They aren't always the starter chests, but they make sense if you've played vanilla. Not in Kansas doesn't tell me anything about where this is